README.md

```diff
@@ -1,2 +1,131 @@
+# Rails With React
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Database](#database)
+  - [Usage](#usage)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Hello Rails Backend] <a name="about-project"></a>
+
+The **Hello Rails** is a Ruby on Rails backend API.
+
+## 🛠 Built With <a name="built-with"></a>
+
+Ruby, Ruby on Rails, Postgresql, Visual Studio Code, React Redux
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby on Rails</summary>
+</details>
+
+<details>
+  <summary>PostgreSQL</summary>
+</details>
+
+### Key Features <a name="key-features"></a>
+- **[Database Connection]**
+- **[Rails API]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites <a name="prerequisites"></a>
+- Ruby
+- Ruby on Rails
+- PostgreSQL
+- Git
+- React
+
+### Setup <a name="setup"></a>
+
+To use this project:
+```
+git clone https://github.com/Xghayor/hello-rails-backend.git
+cd hello-rails-backend
+
+```
+
+### Install <a name="install"></a>
+
+To install the project's dependencies, run:
+
+```
+bundle install
+```
+
+### Database <a name="database"></a>
+
+```
+rails db:create
+rails db:migrate
+rails db:seed
+```
+
+### Usage <a name="usage"></a>
+
+To use, run:
+
+```
+rails s
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Fron-end Link <a name="contributing"></a>
+
+- [Frontend](https://github.com/Xghayor/hello-rails-frontend).
+
+## 👥 Author <a name="author"></a>
+
+👤 **Ghayoor Ul Baqir**
+- GitHub: [@Ghayor](https://github.com/Xghayor)
+- LinkedIn: [GhayoorUlBaqir](https://www.linkedin.com/in/ghayoor-ul-baqir/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Extend the Functionality]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Xghayor/hello-rails-backend/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Kindly give this project a star if you like it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSE) license.
```

Comment:
- [ ] Please make sure that your `README` file doesn't have any broken links because having broken links in your documentation would result in a bad experience for the users, who want to know about the permission or `usage` of this project to either `contribute` or *use* for their projects.

![image](https://github.com/Xghayor/hello-rails-backend/assets/66476329/02f773a5-00cb-4018-afb9-0f8e4e7e8a1b)